iclr_wrap_up/plotter/activations.py

```diff
@@ -25,12 +26,15 @@ def plot(self, measures_summary):
         for layer in range(num_layers):
             ax = fig.add_subplot(num_layers, 1, layer + 1)
 
+            min, max = utils.get_min_max(activations_summary, layer_number=layer)
```

Comment:
min, max are builtin functions that should not be overridden. maybe min_activation, max_activation